.github/workflows/ci.yml

```
@@ -0,0 +1,113 @@
+          echo "GITHUB_ACTION=$GITHUB_ACTION"
+          echo "GITHUB_ACTIONS=$GITHUB_ACTIONS"
+          echo "GITHUB_ACTOR=$GITHUB_ACTOR"
+          echo "GITHUB_REPOSITORY=$GITHUB_REPOSITORY"
+          echo "GITHUB_EVENT_NAME=$GITHUB_EVENT_NAME"
+          echo "GITHUB_EVENT_PATH=$GITHUB_EVENT_PATH"
+          echo "GITHUB_WORKSPACE=$GITHUB_WORKSPACE"
+          echo "GITHUB_SHA=$GITHUB_SHA"
+          echo "GITHUB_REF=$GITHUB_REF"
+          echo "GITHUB_HEAD_REF=$GITHUB_HEAD_REF"
+          echo "GITHUB_BASE_REF=$GITHUB_BASE_REF"
+          echo "::debug::---Start content of file $GITHUB_EVENT_PATH"
+          cat $GITHUB_EVENT_PATH
+          echo "\n"
+          echo "::debug::---end"
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get install -y autoconf \
+                              automake \
+                              autotools-dev \
+                              checkbot \
+                              dia \
+                              docbook \
+                              docbook-xml \
+                              docbook-xsl \
+                              docbook-xsl-ns \
+                              docbook-xsl-saxon \
+                              docbook5-xml \
+                              fop \
+                              ghostscript \
+                              imagemagick \
+                              inkscape \
+                              jing \
+                              libimage-exiftool-perl \
+                              libsaxon-java \
+                              libtool \
+                              libxml-commons-external-java \
+                              libxml2 \
+                              libxml2-utils \
+                              libxmlgraphics-commons-java \
+                              libxslt1.1 \
+                              optipng \
+                              poppler-utils \
+                              python-libxml2 \
+                              python-lxml \
+                              python3 \
+                              python3-lxml \
+                              remake \
+                              sgml-base \
+                              sgml-data \
+                              shtool \
+                              shunit2 \
+                              transfig \
+                              w3m \
+                              xfig \
+                              xmlstarlet \
+                              xsltproc \
+                              zip
+
+      - uses: actions/checkout@v2
+
+      - uses: actions/checkout@v2
+        with:
+          repository: openSUSE/suse-xsl
+          path: suse
+
+      - name: Configure
+        run: ./configure --sysconfdir=/etc
+
+      - name: make
+        run: make
+
+      - name: make install
+        run: sudo make install
+
+      - name: Running daps XML syntax check for double IDs...
```

Language improvement:

```suggestion
      - name: Running daps XML syntax check for duplicated IDs...
```